website/blog/2021-07-29-Introduction to Flint.md

```diff
@@ -0,0 +1,92 @@
+---
+slug: intro to flint
+title: Introduction to FLINT
+author: Ruchi Pakhle
+author_url: https://github.com/Ruchip16
+author_title: Passionate programmer, coder & a web-developer 
+tags: [Intro to Flint, docusaurus]
+description: Introduction to Flint
+---
+<!-- Summary -->
+
+**The Full Lands Integration Tool (FLINT)** is moja global’s flagship software. FLINT is a modular and highly flexible open-source software technology designed to meet the needs of governments, the private sector and other users in measuring, reporting and verifying (MRV) – or forecasting – greenhouse gas emissions and removals from forestry, agriculture and other land uses (AFOLU).
+ 
+<!--truncate-->
+
+### What is FLINT?
+FLINT stands for (Full Lands Integration Tool) which is a C++ framework that combines the data acquired from the satellites & ground in a way that meets the requirements of the policy. FLINT is an *open-source library* maintained under **moja-global**, a project under the **LINUX Foundation** 
+
+### Why FLINT?
+FLINT enables all countries to develop and operate advanced systems. It is based on more than 20 years of experience in building and operating integrated tools in Australia and Canada. It is a universal platform with a modular structure, allowing countries to attach any kind of models or data to build a country-specific system. The platform handles complex computer science tasks, such as storing and processing large data sets, allowing users to focus on monitoring, reporting, and scenario analysis.
+
+### What distinguishes FLINT from other integrating tools?
+FLINT integrates data from remote sensing, ground observations, and other sources to estimate fluxes and stocks of greenhouse gasses in different pools consistent with the guidance from the Intergovernmental Panel on Climate Change (IPCC).
+
+FLINT is using the lessons learned from the first generation of tools to build a new framework that meets current and future needs. Compared with the first-generation tools, the main improvements include:
+
+- A flexible and modular approach that allows specific countries from IPCC Tier 1 to 3 to implement a comprehensive, sample-based, jurisdictional, and supply chain-based approach.
+- The ability to cover all land uses and land-use changes, and activity-based reporting such as REDD+.
+- Scenario analysis systems to allow the development of projections.
+- Potential for a variety of other purposes, including economics, water, and biodiversity.
+- Development is managed using a true open-source approach under moja global, which will allow users (countries, companies, and organizations) to direct strategy and control the budget.
+- Software that allows data processing on local desktops or cloud-based systems.
```

Comment:
Taken directly from [here](https://github.com/moja-global/FLINT#how-is-flint-different-from-earlier-integrating-tools).